src/ngraph/coordinate.hpp

```diff
@@ -75,13 +75,19 @@ namespace ngraph
 
     // Removes some values from a vector of axis values
     template <typename AXIS_VALUES>
-    AXIS_VALUES project(const AXIS_VALUES& axis_values, const AxisSet& deleted_axes)
+    AXIS_VALUES project(const AXIS_VALUES& axis_values,
```

Comment:
I'd rather this be two methods, `project` (with the new semantics) and `remove` or as a renaming of the current one. `remove` could just call `project` with the correct arguments.